libcontainer/configs/validate/validator.go

```diff
@@ -150,6 +150,7 @@ func (v *ConfigValidator) sysctl(config *configs.Config) error {
 	)
 
 	for s := range config.Sysctl {
+		s := strings.Replace(s, "/", ".", -1)
```

Comment:
I think there is a problem with your conversion:
If the ssyctl value is net/pv4/conf/eno2.100/rp_filter
The correct conversion result should be net.ipv4.conf.eno2/100.rp_filter

see kubernetes/kubernetes#102393 (comment)

Reply:
@mengjiao-liu Could you open a github issue?

Reply:
Ok, I will open a issue.